### DISCUSSION TITLE
oven-sh / bun really ?

### DISCUSSION THREAD

**Discussion by LLyaudet** [oven-sh/bun#11507] [General]
### What version of Bun is running?

no one

### What platform is your computer?

Ubuntu

### What steps can reproduce the bug?

Of course, the bug next to "Issue: 🐛 Bug Report" is purple. I've lost some time on the Internet, but fortunately, I never searched and saw a "bun in the oven" video, unless it was an image added "à la Fight Club" or I was not conscious while regarding. What is "sh" for ? shut ? shè ? A married guy with child... at the laboratory at E.N.S. Lyon tested me when I was a PhD doctorate with a load of horrible things you can see on the Internet. He mentioned things like "Bun in the oven" and other things almost no one should want to see. I was like "Ok, I'm a single guy using Internet for... but I never saw such things and I have to learn that such shits exist by a married guy". I'm nice because I still think that the guy was testing me and not enjoying such videos. May I ask what is the goal of this project with oven-sh account and bun repository under this account ? I don't want to learn what is hidden here. I'm not expecting testing anymore, at least testing that I can bear. 

### What is the expected behavior?

No such things as "Bun in the oven".
Regarding your repository, I don't know. I'll let more qualified people look and judge. There are people whose job is to do that.

### What do you see instead?

I don't know, maybe DEATH.

### Additional information

Pray.

**Comment by emastho** [comment]
#random chat leaked a member

**Reply by 7f8ddd** [reply]
> #random chat leaked a member

![image](https://github.com/oven-sh/bun/assets/141754874/bea256dc-110d-4f87-a373-7f5a60eb098c)
